### PR TITLE
add travis

### PR DESCRIPTION
**Pull request by wangqj**

### CODE REVIEW

File: spring-cloud-servicecomb-discovery/src/main/java/org.springframework.cloud.servicecomb.discovery/client/ServiceCombClient.java [hunk 1]
```diff
@@ -51,17 +57,82 @@
 
   private String url;
 
+  private List<String> serviceCenterList = new ArrayList<>();
```
Comment (DouBean) [inline spring-cloud-servicecomb-discovery/src/main/java/org.springframework.cloud.servicecomb.discovery/client/ServiceCombClient.java]: 是否有并发调用？

File: spring-cloud-servicecomb-discovery/src/main/java/org.springframework.cloud.servicecomb.discovery/client/util/URLUtil.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  public static String transform(String restUrl, String scheme) {
+    if (restUrl == null) {
+      return null;
+    }
+    return scheme + SCHEMA_SEPRATOR
+        + restUrl.substring(restUrl.indexOf(SCHEMA_SEPRATOR) + SCHEMA_SEPRATOR.length());
+  }
+
+  public static boolean isEquals(String url1, String url2) {
+    String url1ComparePart = url1.substring(url1.indexOf(SCHEMA_SEPRATOR) + SCHEMA_SEPRATOR.length());
```
Comment (DouBean) [inline spring-cloud-servicecomb-discovery/src/main/java/org.springframework.cloud.servicecomb.discovery/client/util/URLUtil.java]: 参数校验？

File: spring-cloud-servicecomb-discovery/src/main/java/org.springframework.cloud.servicecomb.discovery/client/ServiceCombClient.java [hunk 3]
```diff
@@ -51,17 +57,82 @@
...
+    }
+    return null;
+  }
+
+  public MicroserviceInstanceResponse getServiceCenterInstances()
+      throws RemoteOperationException {
+    Response response = null;
+    try {
+      response = httpTransport.sendGetRequest(buildURI("/registry/health"));
+      if (response.getStatusCode() == HttpStatus.SC_OK) {
```
Comment (DouBean) [inline spring-cloud-servicecomb-discovery/src/main/java/org.springframework.cloud.servicecomb.discovery/client/ServiceCombClient.java]: response  NullPointerException

Comment (wangqj) [reply spring-cloud-servicecomb-discovery/src/main/java/org.springframework.cloud.servicecomb.discovery/client/ServiceCombClient.java]: Response resp = new Response();

File: spring-cloud-servicecomb-discovery/src/main/java/org.springframework.cloud.servicecomb.discovery/client/util/URLUtil.java (hunk 2 above)
Comment (wangqj) [reply spring-cloud-servicecomb-discovery/src/main/java/org.springframework.cloud.servicecomb.discovery/client/util/URLUtil.java]: done

File: spring-cloud-servicecomb-discovery/src/main/java/org.springframework.cloud.servicecomb.discovery/client/ServiceCombClient.java (hunk 1 above)
Comment (wangqj) [reply spring-cloud-servicecomb-discovery/src/main/java/org.springframework.cloud.servicecomb.discovery/client/ServiceCombClient.java]: no

**Pull request merged**